vllm/engine/llm_engine.py

```diff
@@ -1742,14 +1780,41 @@ def _get_stats(self,
                     max_num_generation_tokens_requests.append(
                         max(seq.get_output_len()
                             for seq in seq_group.get_seqs()))
+                    total_tokens_in_current_batch_requests.append(
+                        total_tokens_in_current_batch)
                     if seq_group.sampling_params is not None:
                         n_requests.append(seq_group.sampling_params.n)
                         max_tokens_requests.append(
                             seq_group.sampling_params.max_tokens)
+                        # Update max token capacity as prompt tokens +
+                        # max generation tokens
+                        max_token_capacity = len(
+                            seq_group.prompt_token_ids
+                        ) + seq_group.sampling_params.max_tokens
+                        seq_group.metrics.max_token_capacity = (
+                            max_token_capacity)
+                    max_token_capacity_requests.append(max_token_capacity)
```

Comment:
This will crash if `seq_group.sampling_params is None` 😱

Reply:
The implementation is to be updated shortly.